test/conftest.py

```diff
@@ -421,10 +421,13 @@ def rauc_dbus_service_with_system_adaptive(tmp_path, dbus_session_bus, create_sy
 
 
 class Bundle:
-    def __init__(self, tmp_path, name):
+    def __init__(self, tmp_path, name=None):
```

Comment:
Wouldn't `[..], name="test")` be more simple here?

Reply:
With `=None`, the function itself can distinguish between the default and explicit cases, which is often useful to when refactoring.